### DISCUSSION TITLE
People id for avatar identify

### DISCUSSION THREAD

**Discussion by ClarkDuck** [mozilla/hubs#5056] [Avatar Questions]
Hi 

How can I detect people id in the avatar related code? likely in the [animation-mixer.js](https://github.com/mozilla/hubs/blob/master/src/components/animation-mixer.js) or [player-info.js](https://github.com/mozilla/hubs/blob/master/src/components/player-info.js).
I tried to use the avatarId or displayName from the profile information that provided at this [code](https://github.com/mozilla/hubs/blob/6d3d3392e2b6c1b7bb37a05669a44afe22573080/src/react-components/room/PeopleSidebarContainer.js#L87), but it was ot enough.
I need the constant people id when people changes his/her avatar or his/her name.
And also need this id when the avatar model is created/updated in the [animation-mixer.js](https://github.com/mozilla/hubs/blob/master/src/components/animation-mixer.js) or [player-info.js](https://github.com/mozilla/hubs/blob/master/src/components/player-info.js).

Thank in avdance!

**Comment by pattersonbl2** [comment]
you could get the account id from the admin panel related to each user's email address.  using this account id, you could look in the database for related values for that user's account.

**Reply by ClarkDuck** [reply]
Thank you for your reply.
Btw, how can get this id when the avatar model is creating?

**Reply by ClarkDuck** [reply]
Also, if user enter into the room without sign, the user id is generated temporarly. in this case how can get the user id?
